plato/draw/fresnel/FresnelPrimitive.py

```diff
@@ -1,14 +1,38 @@
 import fresnel
 
-class FresnelPrimitive:
-    def __init__(self, *args, material=None, **kwargs):
-        if material is None:
-            self._material = fresnel.material.Material(solid=0,
-                                                       color=(0, 0, 0),
-                                                       primitive_color_mix=1,
-                                                       roughness=0.3,
-                                                       specular=0.5,
-                                                       spec_trans=0,
-                                                       metal=0)
-        else:
-            self._material = material
+DEFAULT_MATERIAL = fresnel.material.Material(solid=0,
+                                             color=(0, 0, 0),
+                                             primitive_color_mix=1,
+                                             roughness=0.3,
+                                             specular=0.5,
+                                             spec_trans=0,
+                                             metal=0)
+
+SOLID_MATERIAL = fresnel.material.Material(solid=1,
+                                           color=(0, 0, 0),
+                                           primitive_color_mix=1,
+                                           roughness=0.3,
+                                           specular=0.5,
+                                           spec_trans=0,
+                                           metal=0)
+
+
+class FresnelPrimitive(object):
+    """A mixin class that defines a default :py:class:`fresnel.material.Material`.
+    """
+
+    def __init__(self, *args, **kwargs):
+        self._material = kwargs.get('material', DEFAULT_MATERIAL)
+        super().__init__(*args, **kwargs)
+
+
+class FresnelPrimitiveSolid(FresnelPrimitive):
```

Comment:
I created a new mixin class that is used for 2D primitives, which need to have `solid=1` in their material definition to prevent things (e.g. disks, which are actually created as spheres) from looking 3D.